.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+
+defaults: &defaults
+  docker:
+    - image: canonicalwebteam/dev
+  working_directory: /srv
+
+jobs:
+  lint-python:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Lint webapp with Flake8 and black
+          command: flake8 canonicalwebteam tests && black --line-length 79 --check canonicalwebteam tests
+  test-python:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            apt install -yy python3-venv
```

Comment:
why do you need this dependency?
We already install python3 in the canonicalwebteam/dev image